webqr/stuss2021_contact.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCi5h0gf93Tf/tByITWE3z5U9u6sSevQ9OjFUg7AwvQF2RSj6KE4b3qgnm3dkyJr27zTHwNaK26pzzYqI8EwKzDQO9IIk06axbPZYcfdwb/4/DrT/x6jLrMrO35LoTt9EcernLf5g2j0qhzkbbvdCgVY8lwU7nkRYEJN16IVVQ2IH1XrBxDdDWp2AsXkmiFh2ckin3w/uigcA0fcu+PG71sGKOPZ3TXWurXd+ATUIilLFHsneK8xZ+OMxBNP3ypQCG/yeJh28pulwYd7n9WvwhOPLjAourxRRNk5gjTgqbCxdAgfhHPfZ1THb77q2cdMXyqqZmcfP2lZeV/k7DpRdz5 Stuss am Fluss 2021 COVID-19 Kontaktverfolgung
```

Comment:
Let's include that on a different path maybe? It's required by the `webui` as well.

Reply:
Okay. Where should we put it? As for `webqr`, the path is hardcoded in the Makefile.

Reply:
Just in the parent folder would be fine for me, or a separate folder `config/`.